README.md

```diff
@@ -1,98 +1,187 @@
-# About QGIS
+<img src="images/README-md/main_logo.png" width="300">
 
 [![Tests](https://github.com/qgis/QGIS/workflows/QGIS%20tests/badge.svg)](https://github.com/qgis/QGIS/actions?query=workflow%3A%22QGIS+tests%22)
 [![Docker Status](https://img.shields.io/docker/automated/qgis/qgis.svg)](https://hub.docker.com/r/qgis/qgis/tags)
 [![Build Status](https://dev.azure.com/qgis/QGIS/_apis/build/status/qgis.QGIS?branchName=master)](https://dev.azure.com/qgis/QGIS/_build/latest?definitionId=1&branchName=master)
 ![Windows cross build with MXE](https://github.com/qgis/QGIS/workflows/Windows%20cross%20build%20with%20MXE/badge.svg)
 
-QGIS is an Open Source Geographic Information System. The project was born in
-May of 2002 and was established as a project on SourceForge in June of the same
-year. We've worked hard to make GIS software (which is traditionally expensive
-commercial software) a viable prospect for anyone with basic access to a
-Personal Computer.  QGIS currently runs on most Unix platforms (macOS/OS X included)
-and Windows. QGIS is developed using the [Qt toolkit](https://qt.io) and C++.  This
-means that QGIS feels snappy to use and has a pleasing, easy to use graphical
-user interface.
-
-QGIS aims to be an easy to use GIS, providing common functions and
-features. The initial goal was to provide a GIS data viewer. QGIS has
-reached that point in its evolution and is being used by many for their
-daily GIS data viewing and editing needs. QGIS supports a number of raster
-, vector and mesh data formats, with new support easily added using the plugin
-architecture.
+# About QGIS
```

Comment:
How about using https://freelance-tech-writer.github.io/table-of-contents-generator/index.html to generate a table of contents at the beginning, as done in the install.md file?

Reply:
Good idea. Let's get this pull committed and do this in next round. I'm hoping people come up with even better visual examples to rotate in too.

Reply:
@benhur07b tells me he explored this, but it pushes the features section further down the page. Think our goal is to get that and the visuals in front of eyeballs as close to the top as possible.

Reply:
I didn't try myself but maybe it depends on how deep we want to expand the toc